core/src/main/java/in/testpress/database/entities/ExamModification.kt

```diff
@@ -0,0 +1,14 @@
+package `in`.testpress.database.entities
+
+import java.text.SimpleDateFormat
+import java.util.*
+
+data class ExamModification(
+    val id: Long,
+    val examDataModifiedOn: String?
+) {
```

Comment:
Why is this class needed? Why can't we have modified date field in Exam model itself?